locales/en/plugin__kubevirt-plugin.json

```diff
@@ -1067,6 +1067,7 @@
   "seconds": "seconds",
   "Secret name": "Secret name",
   "Secret name must be unique in this namespace.": "Secret name must be unique in this namespace.",
+  "Secret name must not contain dots": "Secret name must not contain dots",
```

Comment:
The proper name for the character is "period", so I think we should change "dots" to "periods" here.

Reply:
@upalatucci

Reply:
@metalice done